azurerm_logic_app/outputs.tf

```diff
@@ -0,0 +1,31 @@
+output "id" {
+  value = azurerm_logic_app_workflow.logic_app.id
+}
+
+output "name" {
+  value = var.name
```

Comment:
```suggestion
  value = azurerm_logic_app_workflow.logic_app.name
```

Reply:
The real name of the resource is the one assigned to the resource. In your case resource_name

Reply:
yes, sorry.. fixed